plugins/module_utils/network/ios/rm_templates/bgp_address_family.py

```diff
@@ -2514,41 +2531,53 @@ def __init__(self, lines=None, module=None):
             "name": "redistribute.ospfv3",
             "getval": re.compile(
                 r"""
-                \s\sredistribute\sospfv3\s(?P<process_id>\S+)
-                (\s(?P<type_1>1))?
-                (\s(?P<type_2>2))?
-                (\s(?P<external>external))?
-                (\s(?P<internal>internal))?
-                (\s(?P<nssa_external>nssa-external))?
+                \s+redistribute\sospfv3\s(?P<process_id>\S+)
                 (\smetric\s(?P<metric>\d+))?
+                (\smatch)?
+                (\s(?P<internal>internal))?
+                (\s(?P<ext_type_1>external\s1))?
```

Comment:
As said we would add fresh attributes with correct options in argspec, we would never show the deprecated attributes in facts but rather handle the option flowing in from the playbook in the config side code, and just render back the correct attributes back in facts.